Big-Spender/readme.md

```diff
@@ -68,39 +68,39 @@ INSERT YOUR QUERY HERE
 **You:** Then here's the query for that:
 
 ```sql
-INSERT YOUR QUERY HERE
+SELECT * FROM spends WHERE description ILIKE '%fee%' OR description ILIKE '%fees%';
 ```
 
 **Farnoosh:** Hi, it's me again. It turns out we also need the transactions that have the expense area of 'Better Hospital Food'. Can you help us with that one?
 
 **You:** No worries. Here's the query for that:
 
 ```sql
-INSERT YOUR QUERY HERE
+SELECT * FROM spends s JOIN expense_areas e ON (s.expense_area_id = e.id) WHERE e.expense_area = 'Better Hospital Food';
 ```
 
 **Claire:** Great, that's very helpful. How about the total amount spent for each month?
 
 **You:** You can get that by using the GROUP BY clause. Here's the query:
 
 ```sql
-CREATE YOUR QUERY HERE
+SELECT to_char(date, 'YYYY-MM') AS month, sum(amount) FROM spends GROUP BY month;
```

Comment:
why `YYYY-MM`, what about using to_char and MONTH? (just a thought!) there are many ways this can be done.

Reply:
Thanks for the advice Karley :)  I found many ways
DATE_PART('MONTH',date)
EXTRACT('MONTH' FROM date)
DATE_TRUNC('MONTH',date)